code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -186,7 +188,11 @@
 	M.regenerate_icons()
 	M.update_body()
 
-/datum/game_mode/proc/prepare_syndicate_leader(datum/mind/synd_mind, nuke_code)
+	if(the_bomb)
+		synd_mind.store_memory("<B>Syndicate [the_bomb.name] Code</B>: [the_bomb.r_code]", 0, 0)
```

Comment:
`store_memory()` doesn't seem to have any other arguments.

https://github.com/ParadiseSS13/Paradise/blob/f8966c92f49ac57826e13d91c518b58602e48b4e/code/datums/mind.dm#L118-L119
```suggestion
		synd_mind.store_memory("<B>Syndicate [the_bomb.name] Code</B>: [the_bomb.r_code]")
```

Reply:
And this